qa/standalone/misc/test_failure.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
Actually, a test in misc that deliberately fails for testing purposes was run.  I ignore that test in run-standalone.sh, but teuthology must be running it using another mechanism.